hack/move-upstream.sh

```diff
@@ -18,6 +18,8 @@ repo="${UPSTREAM_REPO:-k8s.io/kubernetes}"
 package="${UPSTREAM_PACKAGE:-pkg/api}"
 
 patch="${TMPDIR}/patch"
+rm -rf "${patch}"
+mkdir -p "${patch}"
 relativedir="../../../${repo}"
```

Comment:
ah ha!  I wondered why this script never worked for me.  Why must this directory be relative?